dysnix/bsc/values.yaml

```diff
@@ -303,6 +303,39 @@ metrics:
       cpu: 500m
       memory: 256Mi
 
+metricsExtra:
+  enabled: false
+  port: 9369
+  portName: metricsextra
+  interval: 3s
+  path: /metrics
+  scrapeTimeout: 3s
+  args: []
+    # - --url
+    # - http://127.0.0.1:9369
+    # - --addr
+    # - ":9369"
+  image:
+    repository: us-docker.pkg.dev/rpcfast/geth-timestamp-diff-exporter/geth
```

Comment:
WDYT on 
* creating public project on dysnix github with the code of this exporter
* push image to github container registry
* configure corresponding CI/CD optionally
 ?